recipes/mender-app-update-module/debian-master/rules

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/make -f
+
+%:
+	dh $@
+
+override_dh_auto_build:
+	true
```

Comment:
Why do we need to override this one? It should work (iow do nothing) as long as there is a Makefile

Reply:
fixed